src/backend/backend.ts

```diff
@@ -92,13 +95,25 @@ const update = () => {
 
 const subject = new Subject<void>();
 
+const subscriptions = new Array<Subscription>();
+
 const bind = (root: DebugElement) => {
+  if (root.injector == null) {
+    // In AoT-enabled debug builds, the data we get back from {@link ng.probe}
```

Comment:
This is not completely true. A user has to do both:
1) use AOT in debug mode and
2) NOT call `enableProdMode()`

Then this data is filled, even with AOT.